spread.yaml

```diff
@@ -227,26 +227,28 @@ exclude:
     - "*.a"
 
 prepare-each: |
-    # systemd on 14.04 does not know about --rotate or --vacuum-time.
-    if [[ "$SPREAD_SYSTEM" != ubuntu-14.04-* ]]; then
-        journalctl --rotate
-        sleep .1
-        journalctl --vacuum-time=1ms
-    else
-        # Force a log rotation with small size
-        sed -i.bak s/#SystemMaxUse=/SystemMaxUse=1K/g /etc/systemd/journald.conf
-        systemctl kill --kill-who=main --signal=SIGUSR2 systemd-journald.service
+    if [ ${SKIP_SETUP:-} != "yes" ]; then
+        # systemd on 14.04 does not know about --rotate or --vacuum-time.
+        if [[ "$SPREAD_SYSTEM" != ubuntu-14.04-* ]]; then
+            journalctl --rotate
+            sleep .1
+            journalctl --vacuum-time=1ms
+        else
+            # Force a log rotation with small size
+            sed -i.bak s/#SystemMaxUse=/SystemMaxUse=1K/g /etc/systemd/journald.conf
+            systemctl kill --kill-who=main --signal=SIGUSR2 systemd-journald.service
 
-        # Restore the initial configuration and rotate logs
-        mv /etc/systemd/journald.conf.bak /etc/systemd/journald.conf
-        systemctl kill --kill-who=main --signal=SIGUSR2 systemd-journald.service
+            # Restore the initial configuration and rotate logs
```

Comment:
are those tabs instead of spaces?

Reply:
Indeed, sorry about that. I'll fix this.